### DISCUSSION TITLE
how do i use addons on the offline editor?

### DISCUSSION THREAD

**Discussion by teddblue** [ScratchAddons/ScratchAddons#3565] [Help] [answered]
im using offline and wondering how to get addons for it

**Comment by Samq64** [comment, marked as answer]
Use [TurboWarp desktop](https://desktop.turbowarp.org) instead, which has editor addon support. I don't think Scratch desktop can support Scratch Addons because it's a **browser** extension while Scratch desktop is an election **app**.
